### PR TITLE
experiment: force Preshuffle=True for decode paged MQA-logits

### PR DESCRIPTION
**Pull request by amd-sriram**
## Summary

Changes the decode paged MQA-logits call in `rocm_fp8_paged_mqa_logits` (`vllm/v1/attention/ops/rocm_aiter_mla_sparse.py`) to pass `Preshuffle=True` unconditionally instead of gating on `block_size > 1`.

This is an experimental A/B to exercise the preshuffle paged-logits kernel path independent of the KV-cache block size.

## Change

```diff
-                Preshuffle=block_size > 1,
+                Preshuffle=True,
```

## Scope / caveats

- Affects only the decode paged path (`deepgemm_fp8_paged_mqa_logits`).
- The aiter kernel asserts `KVBlockSize % 16 == 0` when `Preshuffle=True`; with `KVBlockSize=block_size`, this requires the KV-cache block size to be 16-aligned. With `block_size=1` (per-token page table) the preshuffle contract is not satisfied, so this is intended for block-size-16-aligned KV cache configs only.
- Draft / experimental: for benchmarking, not for merge as-is.

## Test plan

- [ ] Verify server starts with a 16-aligned KV cache block size and the preshuffle kernel is selected.
- [ ] Compare decode correctness and perf vs baseline (`Preshuffle=block_size > 1`).

### CODE REVIEW

**Comment by amd-sriram** [comment]
Closing: this was an experimental A/B for benchmarking only, not intended for merge.
